.github/workflows/doc-builder.yml

```diff
@@ -16,9 +16,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Configure AWS Credentials
-        uses: aws-actions/configure-aws-credentials@v1
+        uses: aws-actions/configure-aws-credentials@8c3f20df09ac63af7b3ae3d7c91f105f857d8497 #v4
         with:
-          role-to-assume: ${{ secrets.CI_AWS_ROLE_ARN }}
```

Comment:
I think we tore this down while performance maintenance on the account it maps to. Let me know if you don't think its appropriate to reuse CI_MAIN_TESTING_ACCOUNT_ROLE_ARN for this use case.